Userland/Utilities/date.cpp

```diff
@@ -43,15 +45,21 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 0;
     }
 
-    // FIXME: this should be improved and will need to be cleaned up
-    // when additional output formats and formatting is supported
-    if (print_unix_date && print_iso_8601 && print_rfc_3339 && print_rfc_5322) {
-        warnln("date: multiple output formats specified");
+    if (print_unix_date + print_iso_8601 + print_rfc_3339 + print_rfc_5322 + !format_string.is_null() > 1) {
```

Comment:
This feels slightly wrong, but I don't know how to express more elegantly that at most 1 of these `bool`s can be true.

Reply:
the integer conversion should work. Using pure boolean logic would look pretty gnarly.
https://stackoverflow.com/a/33268481